server/app/controllers/admin/AdminReportingController.java

```diff
@@ -57,12 +58,11 @@ private CiviFormProfile getCiviFormProfile(Http.Request request) {
 
   @Secure(authorizers = Authorizers.Labels.ANY_ADMIN)
   public Result show(Http.Request request, String programSlug) {
```

Comment:
suggestion (non-blocking) Since you're updating this anyway I recommend getting rid of manual call to join. Github won't let me give you a clean option to replace because it spans unchanged lines, but this should be all you need to swap the full method.


```java
  public CompletionStage<Result> show(Http.Request request, String programSlug) {
    return programService
        .getActiveFullProgramDefinitionAsync(programSlug)
        .thenApply(
            programDefinition ->
                ok(
                    adminReportingShowView
                        .get()
                        .render(
                            request,
                            getCiviFormProfile(request),
                            programSlug,
                            programDefinition.adminName(),
                            programDefinition.localizedName().getDefault(),
                            reportingService.getMonthlyStats())));
  }
```

Reply:
Done!